instance/management/commands/instance_redeploy.py

```diff
@@ -276,6 +297,37 @@ def _do_mysql_commands(self, instance):
                     cursor.execute(command)
                 cursor.close()
 
+    def _activate_latest_instance(self, instance):
+        """
+        Activate successful redeployments, including enabling monitoring & setting an active VM DNS record.
+
+        This also deactivates old appservers for each successful instance, and sets appropriate tags.
+        """
+        # Make sure that the latest appserver is running
+        latest_appserver = instance.appserver_set.latest('created')
+        if latest_appserver.status == AppserverStatus.Running:
+
+            # Deactivate currently active servers
+            for appserver in instance.appserver_set.filter(_is_active=True):
+                LOG.info(' - Deactivating %s [%s]', appserver, appserver.id)
+                appserver.is_active = False
+                appserver.save()
+
+            # Activate the latest server
+            LOG.info(' - Activating %s [%s]', latest_appserver, latest_appserver.id)
+            latest_appserver.is_active = True
+            latest_appserver.instance.enable_monitoring()
+            latest_appserver.instance.set_active_vm_dns_records()
```

Comment:
We could avoid having to call these function explicitly here by adding a flag to `make_active()` to disable the LB reconfiguration.  We basically want `make_active()` to do everything except for the LB reconfiguration, since that is the slowest bit.

Reply:
Yep, saw the comment below. Makes sense -- will add a flag.